src/main/java/org/apereo/cas/support/oauth/web/endpoints/OAuth20AuthorizeEndpointController.java

```diff
@@ -245,11 +245,14 @@ protected ModelAndView buildAuthorizationForRequest (
                 .map(String::valueOf)
                 .orElseGet(OAuth20GrantTypes.AUTHORIZATION_CODE::getType)
                 .toUpperCase();
-        val accessType = context
+        var accessType = context
                 .getRequestParameter(OsfCasOAuth20Constants.ACCESS_TYPE)
                 .map(String::valueOf)
-                .orElse(OsfCasOAuth20CodeType.ONLINE.name())
+                .orElse(StringUtils.EMPTY)
```

Comment:
This is an optional change since the value will be overwritten by the next statement anyway.